.ci/.grenrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    "username": "elastic",
+    "repo": "apm-agent-php",
+    "dataSource": "prs",
+    "groupBy": {
+        "Breaking changes": ["breaking", "automation"],
```

Comment:
NIT: for testing purposes only

```suggestion
        "Breaking changes": ["breaking"],
```